assets/arc-0060/auth-schema.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
this schema is never used. Right?

Reply:
yeah, will be removed. It's unnecessary if we just need 32 bytes